[2025-04-05T02:47:01.943+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-02T00:00:00+00:00 [queued]>
[2025-04-05T02:47:01.966+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-02T00:00:00+00:00 [queued]>
[2025-04-05T02:47:01.967+0000] {taskinstance.py:1331} INFO - Starting attempt 2 of 2
[2025-04-05T02:47:02.091+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): load_dataset> on 2025-04-02 00:00:00+00:00
[2025-04-05T02:47:02.101+0000] {standard_task_runner.py:57} INFO - Started process 409 to run task
[2025-04-05T02:47:02.111+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_loading_dag', 'load_dataset', 'scheduled__2025-04-02T00:00:00+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/carga_dato.py', '--cfg-path', '/tmp/tmpbms_e30h']
[2025-04-05T02:47:02.143+0000] {standard_task_runner.py:85} INFO - Job 11: Subtask load_dataset
[2025-04-05T02:47:02.363+0000] {task_command.py:410} INFO - Running <TaskInstance: data_loading_dag.load_dataset scheduled__2025-04-02T00:00:00+00:00 [running]> on host 8fa70ebba0d3
[2025-04-05T02:47:02.552+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_loading_dag' AIRFLOW_CTX_TASK_ID='load_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-04-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-02T00:00:00+00:00'
[2025-04-05T02:47:04.318+0000] {logging_mixin.py:149} INFO - Datos cargados en MySQL
[2025-04-05T02:47:04.325+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-04-05T02:47:04.343+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_loading_dag, task_id=load_dataset, execution_date=20250402T000000, start_date=20250405T024701, end_date=20250405T024704
[2025-04-05T02:47:04.406+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-04-05T02:47:04.439+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
